src/core/utils/fs.cpp

```diff
@@ -453,6 +453,18 @@ QString Utils::Fs::QDesktopServicesDownloadLocation()
 #endif
 #endif
 
+#if defined(Q_OS_WIN)
+#if (QT_VERSION < QT_VERSION_CHECK(5, 0, 0))
+    if (QSysInfo::windowsVersion() <= QSysInfo::WV_XP)  // Windows XP
+        return QDir(QDesktopServices::storageLocation(QDesktopServices::DocumentsLocation)).absoluteFilePath(
+                    QCoreApplication::translate("fsutils", "Downloads"));
+    else
+        return QDir(QDesktopServices::storageLocation(QDesktopServices::HomeLocation)).absoluteFilePath("Downloads");
+#else
+    return QStandardPaths::writableLocation(QStandardPaths::DownloadLocation);
```

Comment:
Nice info.
Then @Chocobo1 can you use `QStandardPaths::writableLocation(QStandardPaths::DownloadLocation);` for all OS's if program is compiled with Qt5?